src/shared/components/plots/MultipleCategoryBarPlot.tsx

```diff
@@ -293,7 +293,7 @@ export default class MultipleCategoryBarPlot extends React.Component<IMultipleCa
 
     @computed get chartDomainPadding() {
         return {
-            [this.countAxis]:this.countAxisDomainPadding,
+            [this.countAxis]: this.props.percentage ? 0 : this.countAxisDomainPadding,
```

Comment:
would it make sense to do this even if the axis is absolute (not percentage?)

Reply:
@alisman I found an example. Seems like we cannot do this to not percentage chart, right?
![image](https://user-images.githubusercontent.com/15748980/56978056-a6173e00-6b44-11e9-8918-3da3df1baaa0.png)

Reply:
@schultzn @jjgao can you comment on the above?  With absolute values, should we always try to show a tick at the top?

Reply:
@alisman I think it's ok not to have a tick for absolute values.